pkg/api/types.go

```diff
@@ -204,6 +204,8 @@ type VolumeSource struct {
 
 // Similar to VolumeSource but meant for the administrator who creates PVs.
 // Exactly one of its members must be set.
+// All PersistentVolumes are volumes, only some Volumes are persistent.  All pointers
```

Comment:
Maybe disregard the last comment here?  I'm crossing the streams and just read your comments on the other PR.  Must acquire mutex comment lock.